src/configuration.ts

```diff
@@ -94,3 +94,9 @@ export const getPrismaUrl = (): string | undefined => {
     const prismaUrl = configuration.get<string>('prismaURL');
     return prismaUrl;
 };
+
+export const getExternalChecksDir = (): string | undefined => {
+    const configuration: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration('checkov');
+    const externalChecksDir = configuration.get<string>('externalChecksDir');
+    return externalChecksDir;
```

Comment:
can this be an empty string and cause issues?
Perhaps return undefined when there is a false value?

Reply:
it is similar to how all other config string values are handled. if it's undefined it is not added to the checkov command